docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -146,6 +150,8 @@ decision.
 
 If you provide a non-trivial change, include evidence that alternatives have
 been discussed on the `Django Forum`_ or |django-developers| list.
+Additionally, ensure that a ticket has been submitted and accepted before
+proceeding with your contribution.
```

Comment:
I'm tempted to say, let's drop this addition here. 

I have two reasons to think this.

Firstly, just below here it says "... ask on the ticket for opinions" which implies there is a ticket

Secondly, I am very tempted to drop this whole "Non-trivial contributions" section all together. 
This was introduced in 407f62fd31689b6158902d3ffdcde64aa27679a7 (13 years ago) and hasn't really changed since. To me, this second classification of "trivial" vs "non-trivial" within the same page makes the docs harder to follow. I also think implying that "small bug fixes" are "trivial" is a little harsh on individuals who found it quite difficult to resolve these. 

So maybe let's not touch this section for now 🤔 
@nessita do you have opinions here?